src/parse/parser.rs

```diff
@@ -688,21 +688,19 @@ impl<'help, 'app> Parser<'help, 'app> {
                 let sc_name = match arg_os.to_str() {
                     Some(s) => s.to_string(),
                     None => {
-                        if self.is_set(AS::StrictUtf8) {
-                            return Err(ClapError::invalid_utf8(
-                                Usage::new(self).create_usage_with_title(&[]),
-                                self.app.color(),
-                            ));
-                        }
-                        arg_os.to_string_lossy().into_owned()
+                        return Err(ClapError::invalid_utf8(
```

Comment:
We don't need the check here because we don't let subcommands with invalid utf8 to be built, right?

Reply:
To be clear, this is an existing check. If you know of another place where we check whether an external subcommand is valid (which is handled differently than a normal subcommand), let me know.   My quick glance didn't reveal one but I'm not an expert on the validation code yet.

Reply:
Actually never mind what I was saying originally.

What I want to know is, shouldn't we check that `AS::AllowInvalidUtf8ExternalSubcommands` is not set here?

Reply:
See below about `allow_invalid_utf8_subcommand_with_allow_external_subcommands`

Reply:
Can you please add info about this to Changelog?